src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -15,6 +15,8 @@
 from DockerUtils import push_one_docker, build_dockers, push_dockers, run_docker, find_dockers, build_docker_fullname, copy_from_docker_image, configuration
 from params import default_config_parameters
 
+CLD_INIT_FILE_MAP = "cloudinit/file_map.yaml"
```

Comment:
Please try to use full name whenever possible. `CLD` doesn't unique identity word `CLOUD`.